web-ng/src/app/components/map/map.component.ts

```diff
@@ -59,9 +59,7 @@ export class MapComponent implements OnInit {
   ngOnInit() {
     this.subscription.add(
       this.routerService.getFeatureId$().subscribe(id => {
-        if (id) {
-          this.focusedFeatureId = id;
-        }
+        this.focusedFeatureId = id || '';
```

Comment:
Let's use `null` instead of `''` to make sure subscribers handle this special case correctly.

Reply:
Done